src/settings.js

```diff
@@ -1022,9 +1022,13 @@ var BINARYEN_SCRIPTS = "";
 
 // Whether to ignore implicit traps when optimizing in binaryen.  Implicit traps
 // are the unlikely traps that happen in a load that is out of bounds, or
-// div/rem of 0, etc. We can reorder them, but we can't ignore that they have
-// side effects, so turning on this flag lets us do a little more to reduce code
-// size.
+// div/rem of 0, etc. With this option set, we assume that loads have no side
+// effects, for example, as the only side effect it can have is the possible
```

Comment:
I think this sentence would be clearer as something like "With this option set, the optimizer assumes that loads cannot trap, and therefore that they have no side effects at all."